ATFConverter/Tokenizer.py

```diff
@@ -7,64 +7,90 @@
 class Tokenizer(object):
 
     def line_tokenizer(self, text):
-    
+        """
+        Takes .txt file, reads it, separates lines, and prints only line numbered lines as list of strings (ignores metadata).
+        Text must be a string.
+        """
         line_output = []
 
-        with open(text, mode = "r+", encoding = "utf8") as File:
+        with open(text, mode='r+', encoding='utf8') as File:
             lines = File.readlines()
-            for no_meta in lines: 
-                if re.match(r'^\d*\.|\d\'\.', no_meta):
-                    try:
-                        line_output.append(no_meta) 
-                        #no_meta = re.subn(r'\n', '', no_meta) # doesn't work.
-                    except KeyError:
-                        line_output.append("ERROR")
+            assert isinstance(text, str), 'Incoming argument must be a string.'
+            for no_metadata in lines:
+                if re.match(r'^\d*\.|\d\'\.', no_metadata):
+                    line_output.append(no_metadata)
             return line_output
 
-    def string_line_tokenizer(self: object, untokenized_string: str, include_blanks=False):
-        """This is just CLTK's line tokenizer. It works just fine for ATF files"""
+    def string_tokenizer(self: object, untokenized_string: str, include_blanks=False):
+        """This is CLTK's line tokenizer. It works just fine for strings."""
         assert isinstance(untokenized_string, str), 'Incoming argument must be a string.'
-
         if include_blanks:
             tokenized_lines = untokenized_string.splitlines()
         else:
             tokenized_lines = [line for line in untokenized_string.splitlines() if line != r'\\n']
         return tokenized_lines
 
-    def sign_tokenizer(self, text):
+    def word_tokenizer(self, line_tokenizer):
```

Comment:
This argument name is deceptive... you're just passing some lines into the function, not an actual tokenizer function.